sensu/sensu-docs
# Re-organize Sensu 2 install docs

## Description

- Change Docker install commands to use `latest` instead of specifying a specific release. This obviates the need to update the install docs for each release.
- Remove the Configuring sensuctl page in favor of:
  - Installation and configuration instructions on the main installation page
  - Create, auto-completion, and getting help content on the sensuctl reference page
  - Redirect alias to sensuctl reference page
  - Link to sensuctl installation instructions on the sensuctl reference page
- De-duplicate nightly release installation instructions in favor of linking to main installation instructions
- Assorted formatting cleanup on the installation page
- Adds link to download sensuctl for Windows to partially address #688 
- Adds link to example agent config for Windows

## Motivation and Context


Making information easy to find and maintain

## Review Instructions

Run locally and verify that the installation page is consistent and platform blocks work correctly.

Content changes:

- Install sensuctl for Windows: https://github.com/sensu/sensu-docs/pull/771/commits/bfe64e8e7e4eed6ecde5623a116745a240c2c2c4
- Example configuration file for Windows agent https://github.com/sensu/sensu-docs/pull/771/commits/b75319da598ffd1fadfa9ca5612e4ea8e595a3fd
- Use `service` instead of `/etc/init.d/`  https://github.com/sensu/sensu-docs/pull/771/commits/d54f95a3b6f60823d0c147dce92c24a590356187
